src/partition.rs

```diff
@@ -227,31 +227,41 @@ pub(crate) fn file_read_partitions(
     let mut parts: BTreeMap<u32, Partition> = BTreeMap::new();
 
     trace!("scanning {} partitions", header.num_parts);
+    debug!("Number of parts {:?}\n\n", header.num_parts);
```

Comment:
The `trace!` line above already cover it, please drop this and enable the relevant log level in the application.

Reply:
Will do!